lms/static/sass/base/_layouts.scss

```diff
@@ -0,0 +1,104 @@
+// base layout styles to support early responsive lms
+// may be discarded later once sass breakpoints are wired in
+
+body.view-incourse {
```

Comment:
Thanks for the in-person 411 on this. we'll keep the ``body`` as part of the selector because we have to. mind add that note, though?